### ISSUE TITLE
Support for Multiple Openers

### ISSUE THREAD

**Issue by ccurzio** [Einstein42/myq-garage#17]
This script doesn't seem to support multiple garage door openers. I have two on my MyQ account (both in the same house), and "status" only pulls for one of them. 

\# /tmp/myq-garage/myq-garage.py status
North Garage Door is Closed. Last changed at Thu 04 May 2017 14:40:45

<img width="840" alt="screen shot 2017-05-04 at 2 51 28 pm" src="https://cloud.githubusercontent.com/assets/13078835/25719963/312fabce-30d9-11e7-990f-b2e65d119a52.png">

**Comment by ccurzio** [comment]
I've done a little bit of research on this and I've found that both garage doors do actually appear in res["Devices"] but for some reason, the for loop is not iterating past the first instance.

**Comment by tonypnode** [comment]
can you manually set the door state to the other door?

**Comment by ccurzio** [comment]
Yes. Both doors can be successfully controlled using open "North Garage Door" and open "South Garage Door". It seems just the "status" command doesn't support multiple doors.
